MANIFEST.in

```diff
@@ -0,0 +1,10 @@
+global-exclude **Tests.py
+
+recursive-include src/engine/SCons/Tool/docbook *
```

Comment:
be nice if we could make this a "sub-package".  I can't quite figure out how though.

Reply:
Yes. Indeed. I'd like to figure that one out.